docs/api/proposed_endpoints.md

```diff
@@ -0,0 +1,176 @@
+[TOC]
+
+# Posts related actions
+
+## Create a post
+
+Create a post of contents.
+
+```
+POST api/posts (requires authentication)
```

Comment:
All endpoints should begin with `/`. This should be `/api/posts`

before `/` we accept the `BASE_URL` is already in place. All endpoints should be in respect to the `BASE_URL`